db/migrate/20210615103220_add_trn_verified_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTrnVerifiedToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :teacher_reference_number_verified, :boolean, null: false, default: false
```

Comment:
Likewise, `users` is a much bigger space than we likely want for this boolean.

Reply:
this is gone now and will be in a different PR